package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "2.5.2",
+  "version": "2.5.3-rc.0",
```

Comment:
does this need to be undone or ok as is?

Reply:
Yes, oops.  Thank you!

Reply:
@nickgraz I've not used this before.  Will this get replaced with the new version upon release in NPM?

Reply:
OK as is, the next version will be `2.5.3`, which is tagged when this is merged in. `2.5.3-rc.0` is the first release candidate for `2.5.3` and the version here will get overwritten when the branch is merged.